service/src/test/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsServiceTest.java

```diff
@@ -91,6 +93,31 @@ void shouldReturnFalseWhenThereAreNoEmptyDates() {
         assertThat(service.hasEmptyDates(caseData)).isFalse();
     }
 
+    @Test
+    void shouldPopulateStandardDirections() {
+        LocalDate hearingDate = LocalDate.of(2099, 6, 1);
+        given(calendarService.getWorkingDayFrom(eq(hearingDate), eq(-2))).willReturn(hearingDate.minusDays(2));
+        given(calendarService.getWorkingDayFrom(eq(hearingDate), eq(-3))).willReturn(hearingDate.minusDays(3));
+
+        List<Element<HearingBooking>> hearings = createHearingBookings(hearingDate.atStartOfDay(),
+            hearingDate.atStartOfDay().plusDays(1));
+
+        CaseData caseData = CaseData.builder().hearingDetails(hearings).build();
+
+        Map<String, List<Element<Direction>>> standardDirections = service.populateStandardDirections(
+            caseData);
+
+        //test data in test/resources/ordersConfig.json
+        List<Element<Direction>> allParties = standardDirections.get(ALL_PARTIES.getValue());
```

Comment:
allPartiesDirections, now assertion 
assertThat(unwrapElements(allParties)).containsExactly(expectedDirections[0])
sound weired that parties are equal directions